infra/package.json

```diff
@@ -1,7 +1,13 @@
 {
   "name": "infra",
+  "scripts": {
```

Comment:
I think we could just do e.g. "review" as ENV and fill in the e.g. op://DevOps/passport-$ENV-secrets/CI/BLA, since we are keeping this naming convention across all the repos. But also, it's great as-is!